### PR TITLE
Updated Config File

### PR DESCRIPTION
**Pull request by akshtshrma24**
Updated the config file to have separate sections, and to include left and right IP addresses.

### CODE REVIEW

**Review by neelp03 — APPROVED** [review]

**Review by ShaivPandya — APPROVED** [review]

File: printer/what.sh [hunk 1]
```diff
@@ -23,25 +23,25 @@ create_and_enable_printer() {
...
+    export LEFT_PRINTER_NAME=$(jq -r '.PRINTING.LEFT.NAME' config/config.json)
+    export LEFT_PRINTER_IP=$(jq -r '.PRINTING.LEFT.IP' config/config.json)                                                                                ─╯
+    create_and_enable_printer $LEFT_PRINTER_NAME $LEFT_PRINTER_IP
+fi
+
+if [ $(cat config/config.json | jq ".PRINTING.RIGHT.ENABLED") = "true"  ]; then
+    export RIGHT_PRINTER_NAME=$(jq -r '.PRINTING.RIGHT.NAME' config/config.json)
+    export RIGHT_PRINTER_IP=$(jq -r '.PRINTING.RIGHT.IP' config/config.json)                                                                             ─╯
+    create_and_enable_printer $RIGHT_PRINTER_NAME $RIGHT_PRINTER_IP
+fi
```
Comment (evanugarte) [inline printer/what.sh]: hard 2 catch this one

```suggestion
if [ $(cat config/config.json | jq ".PRINTING.LEFT.ENABLED") = "true"  ]; then
    export LEFT_PRINTER_NAME=$(jq -r '.PRINTING.LEFT.NAME' config/config.json)
    export LEFT_PRINTER_URL=$(jq -r '.PRINTING.LEFT.LPD_URL' config/config.json)                                                                                ─╯
    create_and_enable_printer $LEFT_PRINTER_NAME $LEFT_PRINTER_URL
fi

if [ $(cat config/config.json | jq ".PRINTING.RIGHT.ENABLED") = "true"  ]; then
    export RIGHT_PRINTER_NAME=$(jq -r '.PRINTING.RIGHT.NAME' config/config.json)
    export RIGHT_PRINTER_URL=$(jq -r '.PRINTING.RIGHT.LPD_URL' config/config.json)                                                                             ─╯
    create_and_enable_printer $RIGHT_PRINTER_NAME $RIGHT_PRINTER_URL
fi
```

**Review by evanugarte — COMMENTED** [review]
since the printer name already exists in the config, why don't we return the name in the config value in `determinePrinterForJob` (`PRINTING.LEFT.NAME`, `PRINTING.RIGHT.NAME`)

then in the `setInterval` we just use whatever `determinePrinterForJob` returns directly in the `exec` command

**Comment by evanugarte** [comment]
also we need to update the config imports in `LEDSignHandler.js`

File: printer/PrintHandler.js [hunk 2]
```diff
@@ -45,14 +42,24 @@ function deleteFile(fileNo) {
...
 }
+
 function determinePrinterForJob() {
-  const randomNumber = Math.random();
-  if (randomNumber < 0.5) {
-    return 'left';
-  }
-  else {
-    return 'right';
+  if(PRINTING.LEFT.ENABLED && PRINTING.RIGHT.ENABLED) {
```
Comment (evanugarte) [inline printer/PrintHandler.js]:
```suggestion
  if (PRINTING.LEFT.ENABLED && PRINTING.RIGHT.ENABLED) {
```

**Review by evanugarte — COMMENTED** [review]
change 
```js
`HP-LaserJet-p2015dn-${printer} ${path}`,
```

to

```js
`${printer} ${path}`,
```

**Review by evanugarte — APPROVED** [review]

**Review by evanugarte — APPROVED** [review]
what does neel think

**Review by neelp03 — APPROVED** [review]
I m with it

**Pull request merged**